src/hypothesis/searchstrategy/strategies.py

```diff
@@ -282,9 +282,9 @@ def condition(x):
                 condition,
                 random=random,
                 settings=settings(
-                    max_shrinks=0,
                     database=None,
                     verbosity=Verbosity.quiet,
+                    phases=[p for p in list(Phase) if p != Phase.shrink],
```

Comment:
set(Phase) - set([Phase.shrink]) ?

May (or may not) need some slight fixup for `phases` to work with a set.
I imagine the only method called is `__contains__` though.